osu.Game/Screens/Backgrounds/BackgroundScreenBeatmap.cs

```diff
@@ -18,23 +18,33 @@ namespace osu.Game.Screens.Backgrounds
     public class BackgroundScreenBeatmap : BackgroundScreen
     {
         /// <summary>
-        /// The amount of blur to apply when full user blur is requested.
+        /// The amount of blur to apply when full gameplay blur is requested.
         /// </summary>
-        public const float USER_BLUR_FACTOR = 25;
+        public const float GAMEPLAY_BLUR_FACTOR = 25;
+
+        /// <summary>
+        /// The amount of blur to apply when full UI blur is requested.
+        /// </summary>
+        public const float UI_BLUR_FACTOR = 25;
 
         protected Background Background;
 
         private WorkingBeatmap beatmap;
 
         /// <summary>
-        /// Whether or not user dim settings should be applied to this Background.
+        /// Whether or not gameplay dim/blur settings should be applied to this Background.
         /// </summary>
-        public readonly Bindable<bool> EnableUserDim = new Bindable<bool>();
+        public readonly Bindable<bool> EnableGameplayDim = new Bindable<bool>();
+
+        /// <summary>
+        /// Whether or not UI blur settings should be applied to this Background.
+        /// </summary>
+        public readonly Bindable<bool> EnableUIBlur = new Bindable<bool>();
 
         public readonly Bindable<bool> StoryboardReplacesBackground = new Bindable<bool>();
 
         /// <summary>
-        /// The amount of blur to be applied in addition to user-specified blur.
+        /// The amount of blur to be applied in addition to the game-specified or UI-specified blur.
         /// </summary>
         public readonly Bindable<float> BlurAmount = new BindableFloat();
```

Comment:
This property is now intensely confusing. If gameplay dim is on, the amount is added onto the existing blur, and if UI dim is on, it *multiplies existing blur*. This is all way too complex to live in its current state.